.vscode/settings.json

```diff
@@ -6,5 +6,20 @@
 	"editor.guides.bracketPairs": "active",
 	"gitlens.advanced.blame.customArguments": [
 		"--ignore-revs-file", "${workspaceRoot}/.git-blame-ignore-revs"
-	]
+	],
+	// ESLint settings:
+	"eslint.workingDirectories": [
+		"tgui/"
+	],
+	"eslint.rules.customizations": [
+		// We really want to fail the CI builds on styling errors,
+		// but it's better to show them as yellow squigglies in IDE
+		// and thus differentiate from the red typescript ones which
+		// are actually hard errors.
```

Comment:
```suggestion
		"paradisegit.comment2": [
			"We really want to fail the CI builds on styling errors,",
			"but it's better to show them as yellow squigglies in IDE",
			"and thus differentiate from the red typescript ones which",
			"are actually hard errors.",
		],
```

Reply:
That is indeed another way to address it.
Do you prefer that to adding `-not -path ".vscode/*"` into `ci.yml`?

Reply:
You could also do that, but I feel like it should still be validated